evaluation_registry/evaluations/share_views.py

```diff
@@ -0,0 +1,199 @@
+from django.http import Http404
+from django.shortcuts import redirect, render
+from django.views.decorators.http import require_http_methods
+
+from evaluation_registry.evaluations.forms import EvaluationCreateForm
+from evaluation_registry.evaluations.models import (
+    Department,
+    Evaluation,
+    EvaluationDepartmentAssociation,
+)
+from evaluation_registry.evaluations.views import (
+    evaluation_dates_view,
+    evaluation_description_view,
+    evaluation_type_view,
+)
+
+
+@require_http_methods(["GET"])
+def before_create_view(request):
+    return render(request, "share-form/before-start.html")
+
+
+@require_http_methods(["GET", "POST"])
+def choose_evaluation_status_view(request):
+    options = Evaluation.Status.choices
+    if request.method == "GET":
+        return render(request, "share-form/evaluation-status.html", {"options": options, "error": False})
+    status = request.POST.get("status")
+    if not status:
+        return render(request, "share-form/evaluation-status.html", {"options": options, "error": True})
+    return redirect("create", page_number=3, status=status)
+
+
+@require_http_methods(["GET", "POST"])
+def evaluation_create_view(request, status):
+    errors = {}
+    departments = Department.objects.all()
+    if request.method == "POST":
+        form = EvaluationCreateForm(request.POST)
+
+        form_complete = request.POST.get("form_complete")
+        selected_departments = request.POST.getlist("departments")
+        selected_lead = request.POST.get("lead_department")
+        department_to_remove = request.POST.get("remove_department")
+
+        if form_complete:
+            if form.is_valid():
+                new_evaluation = form.save()
+                EvaluationDepartmentAssociation.objects.create(
+                    evaluation=new_evaluation, department=form.cleaned_data["lead_department"], is_lead=True
+                )
+                if form.cleaned_data["departments"]:
+                    for department in form.cleaned_data["departments"]:
+                        EvaluationDepartmentAssociation.objects.create(
+                            evaluation=new_evaluation,
+                            department=department,
+                        )
+
+                return redirect("share", uuid=new_evaluation.id, page_number=1)
+            errors = form.errors.as_data()
+
+        if department_to_remove and (department_to_remove in selected_departments):
```

Comment:
```suggestion
        if department_to_remove in selected_departments:
```
`None` is a valid obj to look for?